pow/randomx/sys/build.rs

```diff
@@ -17,7 +17,11 @@ fn main() {
 	let dst = config.build();
 
 	if target.contains("pc-windows-msvc") {
-		println!("cargo:rustc-link-search=native={}/build/Release", dst.display());
+		if env::var("PROFILE").unwrap() == "debug" {
+			println!("cargo:rustc-link-search=native={}/build/Debug", dst.display());
```

Comment:
```suggestion
			println!("cargo:rustc-link-search=native={}/build/Debug", dst);
```

Reply:
I guess that`s just how it is supposed to be used:
`A Path might contain non-Unicode data. This struct implements the Display trait in a way that mitigates that. It is created by the display method on Path.`
https://doc.rust-lang.org/std/path/struct.Display.html

Reply:
Ah you're right!